NEWS.md

```diff
@@ -1,3 +1,7 @@
+# webshot2 (development version)
+
+* Fixed `rmdshot()` when used to screenshot an R Markdown document with `runtime: shiny` or `runtime: shinyrmd` (#52).
```

Comment:
I just merged but then realized that this doesn't quite have the same format as other NEWS items. It should be 

```
* Fixed #52: `rmdshot()` did not work when used to screenshot an R Markdown document with `runtime: shiny` or `runtime: shinyrmd`. (#53)
```

I'll fix on `main`.

Reply:
🤔 but there aren't any other NEWS items in this repo. 

Is this style guidance written down somewhere? It would be helpful, both internally and externally if it were. And along those lines, while I can live with us deviating from the tidyverse style, it'd be easier for external collaborators and new contributors if we adopted the [tidyverse style guidelines](https://style.tidyverse.org/news.html).